testsuite/objects/__init__.py

```diff
@@ -109,6 +109,22 @@ def asdict(self):
         return {"name": self.name, **asdict(self.value)}
 
 
+@dataclass
+class ExtendedProperty(Property):
+    """
+    Dataclass extending Property class adding optional `overwrite` feature
+    used in extended_properties functionality in Identity section.
+    """
+
+    overwrite: Optional[bool] = None
+
+    def asdict(self):
+        """Override `asdict` function."""
```

Comment:
Or maybe something like:
```
"""Extending inherited asdict() to include also new attribute(s) defined here"""
```

(btw this means, some sort of generaral asdict method that covers attributes generally might be handy; something to investigate in future)

Reply:
Changed the comment.
Your review gave me idea for a new function (#230). It wont solve this particular inheritance problem but a different one.